### DISCUSSION TITLE
First Pull Request and How To See Changes

### DISCUSSION THREAD

**Discussion by Hugoonreplit** [zed-industries/zed#8309] [Feature Requests]
# First PR

Hello people, I did my first PR yesterday (#8223), and I think everything is correct in it, I think I'm just waiting approval and I would just like to confirm here if I'm missing anything. My first change was adding the R logo to `assets/icons/file_icons/r.svg` and I updated the code in `assets/icons/file_icons/file_types.json` so .r files could be associated with the R logo. 

# How To See Changes

I'm not very good when it comes to all the terminal stuff and Github, but I try my best. In PR #8272 a C++ icon is requested to be added. In the original Zed app the logo is just some curly braces, but the author of this PR is able to show how his changes would affect Zed, how could I do this?

Btw I think my other PR is correct since we did the same modifications.

Thanks in advance.

**Comment by Moshyfawn** [comment]
[The contribution guide](https://github.com/zed-industries/zed/blob/main/CONTRIBUTING.md#tips-to-improve-the-chances-of-your-pr-getting-reviewed-and-merged) describes how to increase the likelihood of your PR being reviewed and merged. In general, ensure that your PR is small and focused. Additionally, be patient as maintainers may not have the time to review your PR immediately.

Regarding previewing changes, although I understand your inexperience with the terminal, you'll need to start to get the hang of some aspects of it in order to be able to contribute changes that affect the implementation details of the project.

For information on running the project locally, please refer to [the Building Zed locally guide](https://github.com/zed-industries/zed/blob/main/docs/src/developing_zed__building_zed_macos.md).

> [!NOTE]  
> I must point out that you included an icon for a language that Zed does not support. Currently, Zed is not seeking contributions to add support for new languages to the core project, [as stated in the contributing guide](https://arc.net/l/quote/bsqvctxf).
> 
> I suggest that you review [the public roadmap](https://zed.dev/roadmap) and focus on more pressing issues that you can address.

In the meantime, I have run your fork locally and taken a screenshot of the changes you made.

<img width="1392" alt="image" src="https://github.com/zed-industries/zed/assets/16290753/63f68791-9d74-4bd1-8065-3698665f7c15">

**Reply by Hugoonreplit** [reply]
Thank you so much!
